packages/auth/src/app.ts

```diff
@@ -216,6 +216,8 @@ export class App {
       grantRoutes.interaction.finish
     )
 
+    this.publicRouter.del('/interact/:interactId/deny', grantRoutes.del)
```

Comment:
Is this pr just covering this bullet in the [GNAP spec](https://datatracker.ietf.org/doc/html/draft-ietf-gnap-core-protocol#section-4)?

```
   When interacting
   with an RO, the AS can do anything it needs to determine the
   authorization of the requested grant, including:
   ...
   *  allow the RO to deny the requested delegation
   ```